display/d.labels/do_labels.c

```diff
@@ -461,6 +478,8 @@ int scan_ref(char *buf)
         if (xmatch(word1) || ymatch(word1))
             return 1;
         FALLTHROUGH;
+    case EOF:
+        FALLTHROUGH;
```

Comment:
Please remove the line with `FALLTHROUGH`, this is what caused the CI failure and it is not needed.

Reply:
done